library/ssl_tls12_server.c

```diff
@@ -37,6 +37,13 @@
 #include "constant_time_internal.h"
 #include "mbedtls/constant_time.h"
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+#include "psa/crypto.h"
+#define HASH_MAX_SIZE PSA_HASH_MAX_SIZE
+#else /* !defined(MBEDTLS_USE_PSA_CRYPTO) */
+#define HASH_MAX_SIZE MBEDTLS_MD_MAX_SIZE
+#endif /* !defined(MBEDTLS_USE_PSA_CRYPTO) */
```

Comment:
The comments on `#else`/`#endif` reduce readability, and aren't necessary

```suggestion
#define HASH_MAX_SIZE PSA_HASH_MAX_SIZE
#else
#define HASH_MAX_SIZE MBEDTLS_MD_MAX_SIZE
#endif
```

Reply:
Will cleanup the unnecessary comments.